packages/SwingSet/misc-tools/vat-map-from-swingstore.js

```diff
@@ -0,0 +1,155 @@
+// @ts-nocheck
+import '@endo/init';
+import sqlite3 from 'better-sqlite3';
+import fs from 'fs';
+import process from 'process';
+
+// run this like:
+//  node vat-map-from-swingstore.js run-04-swingstore-pruned.sqlite vat-map-04.json 04
+//
+// It will extract data about all vats from that SwingStore DB
+// snapshot, and write it into the .json file. These can be compared
+// with the neighboring vat-map-delta.js . The "04" runID is added to
+// the data for the benefit of quick jq scripts that forget the
+// filename they're looking at.
+
+const [swingstoreDBfn, outputFn, runID] = process.argv.slice(2);
+const ssdb = sqlite3(swingstoreDBfn);
+
+const sqlGet = ssdb
+  .prepare('SELECT value FROM kvStore WHERE key = ?')
+  .pluck(true);
+const get = key => sqlGet.get(key);
+const getJSON = key => JSON.parse(get(key));
+const sqlGetRange = ssdb.prepare(
+  'SELECT * FROM kvStore WHERE key > ? AND key < ?',
```

Comment:
```suggestion
  'SELECT * FROM kvStore WHERE key >= ? AND key < ?',
```
(no-op in this file but conforms with our conventions elsewhere)